web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -507,6 +507,8 @@
 
         return self;
       };
+	  
+      self['catch'] = self.fail;
```

Comment:
Given that we haven't heard back from @serveace on this since we commented, I'm not inclined to hold up the merge on a minor stylistic detail. WDYT, @Shadowfiend?